rellic/AST/Z3ConvVisitor.cpp

```diff
@@ -790,13 +847,36 @@ void Z3ConvVisitor::VisitBinaryApp(z3::expr z_op) {
       }
     } break;
 
+    case Z3_OP_XOR:
```

Comment:
There is also BXOR, I am not too entirely familiar with Z3, so I assume I want BXOR, which is what https://github.com/lifting-bits/rellic/pull/52/files#diff-0ad1217b54cf74a0a8b80e2ac5ac98b6R588 emits?

Reply:
Yup, `BXOR` is what you want here. The `B` prefix denotes Z3 operations over bitvectors, which you can imagine as C integers of certain width. For example, `uint32_t` would be a `(_ BitVec 32)` in Z3. If you're interested in learning a bit more, you can check out the Z3 [tutorial](https://rise4fun.com/z3/tutorialcontent/guide) and [C++ API](https://z3prover.github.io/api/html/group__cppapi.html).